Project.toml

```
@@ -1,6 +1,6 @@
 name = "OrderedCollections"
 uuid = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
-version = "1.3.3"
+version = "1.3.4"
```

isn't this is a new feature? Not a bug fix.

```suggestion
version = "1.4.0"
```